app_test.go

```diff
@@ -520,10 +520,25 @@ func TestAppStart(t *testing.T) {
 	})
 
 	t.Run("InvokeNonFunction", func(t *testing.T) {
-		app := NewForTest(t, Invoke(struct{}{}))
+		spy := printerSpy{&bytes.Buffer{}}
```

Comment:
nit (not for this diff stack): We might want to swap a `strings.Builder` into `printerSpy` at some point.